test/tests/mesh/quadrilateral/tests

```diff
@@ -2,13 +2,18 @@
   [quad]
     [setup]
       type = RunCommand
-      command = 'bash -c "cd fluid-openfoam && ./Allclean && fluentMeshToFoam constant/quadrilateral.msh && refineMesh -overwrite"'
+      command = 'bash -c "cd fluid-openfoam && ./Allclean && blockMesh"'
```

Comment:
The other tests use `fluentMeshToFoam`, can we not use that here? If not should we remove `quadrilateral.msh`?

Reply:
Ok speaking to Harry and googling, `blockMesh` only does quads. Using `blockMesh` saves us from committing the much larger `.msh` files. I'm happy with this.